test/solidity/Facets/GlacisFacet.t.sol

```diff
@@ -0,0 +1,284 @@
+                bytes32(uint256(uint160(bridgeData.receiver))),
+                bridgeData.destinationChainId,
+                REFUND_WALLET,
+                payableAmount
+            );
+
+        addToMessageValue =
+            quoteSendInfo.gmpFee.nativeFee +
+            quoteSendInfo.airliftFeeInfo.airliftFee.nativeFee;
+
+        // produce valid GlacisData
+        validGlacisData = GlacisFacet.GlacisData({
+            refund: REFUND_WALLET,
+            nativeFee: addToMessageValue
+        });
+    }
+
+    function initiateBridgeTxWithFacet(bool) internal override {
+        glacisFacet.startBridgeTokensViaGlacis{ value: addToMessageValue }(
+            bridgeData,
+            validGlacisData
+        );
+    }
+
+    function initiateSwapAndBridgeTxWithFacet(bool) internal override {
+        glacisFacet.swapAndStartBridgeTokensViaGlacis{
+            value: addToMessageValue
+        }(bridgeData, swapData, validGlacisData);
+    }
+
+    function testBase_CanBridgeNativeTokens() public override {
+        // facet does not support bridging of native assets
+    }
+
+    function testBase_CanBridgeTokens()
+        public
+        override
+        assertBalanceChange(
+            ADDRESS_WORMHOLE_TOKEN,
+            USER_SENDER,
+            -int256(defaultWORMHOLEAmount)
+        )
+        assertBalanceChange(ADDRESS_WORMHOLE_TOKEN, USER_RECEIVER, 0)
+        assertBalanceChange(ADDRESS_DAI, USER_SENDER, 0)
+        assertBalanceChange(ADDRESS_DAI, USER_RECEIVER, 0)
+    {
+        vm.startPrank(USER_SENDER);
+
+        // approval
+        wormhole.approve(address(glacisFacet), bridgeData.minAmount);
+
+        //prepare check for events
+        vm.expectEmit(true, true, true, true, address(glacisFacet));
+        emit LiFiTransferStarted(bridgeData);
+
+        initiateBridgeTxWithFacet(false);
+        vm.stopPrank();
+    }
+
+    function testBase_CanBridgeTokens_fuzzed(uint256 amount) public override {
+        vm.assume(
+            amount > 0 * 10 ** wormhole.decimals() &&
+                amount < 100_000 * 10 ** wormhole.decimals()
+        );
+        vm.startPrank(USER_SENDER);
+        bridgeData.minAmount = amount;
+        // approval
+        wormhole.approve(address(glacisFacet), bridgeData.minAmount);
+        QuoteSendInfo memory quoteSendInfo = IGlacisAirlift(address(airlift))
+            .quoteSend(
+                bridgeData.sendingAssetId,
+                bridgeData.minAmount,
+                bytes32(uint256(uint160(bridgeData.receiver))),
+                bridgeData.destinationChainId,
+                REFUND_WALLET,
+                payableAmount
+            );
+        addToMessageValue =
+            quoteSendInfo.gmpFee.nativeFee +
+            quoteSendInfo.airliftFeeInfo.airliftFee.nativeFee;
+        //prepare check for events
+        vm.expectEmit(true, true, true, true, address(glacisFacet));
+        emit LiFiTransferStarted(bridgeData);
+        initiateBridgeTxWithFacet(false);
```

Comment:
leave blank lines between unconnected code, e.g.

        vm.expectEmit(true, true, true, true, address(glacisFacet));
        emit LiFiTransferStarted(bridgeData);
        
        these two are closely connected...ok to not have a blank line
        
        
        emit LiFiTransferStarted(bridgeData);
        initiateBridgeTxWithFacet(false);
        
        these two are not related at all, blank line pls
        same before vm.stopPrank();